nvflare/app_common/ccwf/swarm_client_ctl.py

```diff
@@ -554,6 +554,9 @@ def do_learn_task(self, name: str, task_data: Shareable, fl_ctx: FLContext, abor
 
         self.log_info(fl_ctx, f"Round {current_round} started.")
 
+        learnable = self.persistor.load(fl_ctx)
+        fl_ctx.set_prop(AppConstants.GLOBAL_MODEL, learnable, private=True, sticky=True)
```

Comment:
Why do you need to do this?

Reply:
Right this isn’t needed here, only for starting client. Making a fix since persistor.load() creates PTModelPersistenceFormatManager, so need to create this somewhere else since load() will only be called at initial site.